osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
@SoykaDaimler I agree to your further definition. Consider adding counterclockwise when in bird-view. The question about reference frames will be clarified in #39